src/screens/toDoList/components/types.ts

```diff
@@ -1,6 +1,14 @@
 import {ToDoListItem} from '../types';
 
 export type ToDoListRowProps = {
+  key: string;
+  toDoListItem: ToDoListItem;
+  deleteItem: () => void;
+  isOpen: boolean;
+  onSwipe: () => void;
+};
+
+export type FlatListToDoListProps = {
```

Comment:
You could use the same type.
When we create a component, we get the key props from React. We don't need to put it in the type